models/base/service.yml

```
@@ -16,6 +16,12 @@ generics:
         label: Name
         optional: false
         order_weight: 1
+      - name: service_identifier
+        kind: NumberPool
+        read_only: true
```

Not related to this PR but I think we shouldn't require the user to define `read_only: true` for NumberPool and Computed Attribute, it should be automatic and we should raise an error if user is trying to enforce `read_only: false`
cc @ogenstad @wvandeun

---

For now as the `read_only` attribute defaults to `False` in the models I'm not sure we can do that. If we look at models for the schema generation we could change it so that it would default to `None` and get changed to `False` if no choice has been made.

---

I do agree that it would be nice to have this type of attribute (and computed attribtute) to be read-only by default. 
If it is not possible to do it today, or if it is to involved the we should open an issue/feature request in the future.

---

I think it would be part of `INFP-234` where we somewhere add a note that we want this.

---

Added the information as a comment on it